middlewares/members.js

```diff
@@ -167,8 +172,14 @@ exports.setUserPassword = async (req, res) => {
     }
 
     await userWithPassword.update({ password: req.body.password });
+
     // TODO: password should be sent to gsuite-wrapper
-    // await superagent.post('gsuite-wrapper:8084/accounts?SETPASS')
+    if (req.currentUser.gsuite_id) {
+        const payload = {
+            password: crypto.createHash('sha1').update(JSON.stringify(req.body.password)).digest('hex')
```

Comment:
github complains here, is it okay? if it is, can it be muted?

Reply:
This needs to be fixed in GSuiteWrapper itself first, we can temporarily mute it if we merge this before